utils/react/test/render-if.test.mjs

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react'
```

Comment:
Should this file be named `render-if.test.jsx` since jsx is being used in this file?

Reply:
we've been following along with what ever the file is named. If the main file ends win `mjs` then the test file should be end in `mjs` even if it does have `jsx`.